src/main.c

```diff
@@ -443,18 +444,22 @@ int main(int argc, char* argv[]) {
         continue;
       }
       if (isoption(argv[i], 0, "argfile", &short_opts) ||
+          isoption(argv[i], 0, "rawfile", &short_opts) ||
           isoption(argv[i], 0, "slurpfile", &short_opts)) {
+        int raw = isoption(argv[i], 0, "rawfile", &short_opts);
         const char *which;
         if (isoption(argv[i], 0, "argfile", &short_opts))
           which = "argfile";
+        else if(isoption(argv[i], 0, "rawfile", &short_opts))
```

Comment:
You don't have to repeat this `isoption()` call here.  You can just `if (raw) ...`.  I'll fix this and push.

Reply:
Thanks!